src/main/java/com/google/sps/servlets/ReviewServlet.java

```diff
@@ -34,7 +34,13 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String place_id = request.getParameter("place_id");
     List<PlaceReviews> allLocations = queryLocation(place_id);
-    PlaceReviews location = trimQuery(allLocations);
+    
+    PlaceReviews location;
+    if (allLocations.size() == 0) {
+      location = null;
+    } else {
+      location = trimQuery(allLocations);
+    }
```

Comment:
May I suggest:
<pre>
<code>
location = (allLocations.size() == 0) ? null : trimQuery(allLocations);
</code>
</pre>
<p> Or, perhaps, if you're feeling extra fancy: </p>
<pre>
<code>
location = (allLocations.size() ) ? trimQuery(allLocations) : null;
</code>
</pre>

Reply:
Wow, this is just the tepid fix I would love.